addons/launcher-template/launcher-template.addon

```diff
@@ -0,0 +1,4 @@
+# Name: launcher-template
+# Description: Imports the launch.openshift.io template
+
```

Comment:
@gastaldi Also in the description please add from which git repo we have those templates. It would be great if you also include tag/release so when a issue  come up regarding template we can easily track it.